thrift/src/test/java/com/linecorp/armeria/it/server/GracefulShutdownIntegrationTest.java

```diff
@@ -143,8 +143,8 @@ public void waitsForRequestToComplete() throws Exception {
         final CountDownLatch latch = new CountDownLatch(1);
         CompletableFuture.runAsync(() -> {
             try {
-                latch.countDown();
                 client.sleep(500L);
+                latch.countDown();
```

Comment:
Because `server.stop()` can be called before `client.sleep(500L)`, I move `latch.countDown()` under the `client.sleep(500L)` so the request is always sent before sutting down.